packages/frontend/src/components/project/Callout.tsx

```diff
@@ -21,7 +21,7 @@ export function Callout({ color, icon, body, className }: CalloutProps) {
         className,
       )}
     >
-      <span>{icon}</span>
+      <span className={cx('flex items-center')}>{icon}</span>
```

Comment:
Changes in this component caused many undesired layout shifts (check out the storybook-visual-regression). IMO, we could use a new simple component with a background and icon, that will be only responsible for the "project archived" warning. No need to complicate the warning bar then.